### DISCUSSION TITLE
Bind Shell Not Working on Server Vulnerable With EternalBlue

### DISCUSSION THREAD

**Discussion by flamecopper** [rapid7/metasploit-framework#16232] [Q&A]
Hi there,

I managed to gain admin access to a computer. I managed to find another network with SMBv1 Eternalblue enabled.
I have added the necessary autoroute feature and managed to find another host with *.80 and SMBv1 vulnerable.
However, when I used bind shell it is not really working. Please see the following image.
![image](https://user-images.githubusercontent.com/1225583/155864825-9122b071-8be8-4f32-a942-2e7b058aeea3.png)
![image (1)](https://user-images.githubusercontent.com/1225583/155864828-e35bc6ff-19d3-4eb1-954a-8c2c0620d01d.png)

**Comment by flamecopper** [comment]
HI appreciate any advise, this is ran over a pivot.
And the server is Windows Server 2012 R2 Datacenter 9600

**Comment by its0x08** [comment]
That is probably because the network device that the box you are attacking is under, has pre-defined rules about which port to forward so the port on the box will open but wont be forwarded so you wont have access remotely.

That is just a speculation, and that is because the information provided is too poor!
More information needed!!
